tomviz/DataPropertiesPanel.h

```diff
@@ -65,11 +68,15 @@ private slots:
   QPointer<DataSource> m_currentDataSource;
   QPointer<pqProxyWidget> m_colorMapWidget;
   QPointer<QWidget> m_tiltAnglesSeparator;
+  DataPropertiesModel m_scalarsTableModel;
 
   void clear();
   void updateSpacing(int axis, double newLength);
-  void updateInformationWidget(QTreeWidget* infoTreeWidget,
-                               vtkPVDataInformation* dataInformation);
+  QList<ArrayInfo> getArraysInfo(vtkPVDataInformation* dataInfo) const;
+  void updateInformationWidget(QTableView* scalarsTable,
```

Comment:
The QList parameters should be passed in as const references too, missed these first time around.